_data/changelogs/component-input-prefix-suffix.yml

```diff
@@ -2,6 +2,19 @@ title: Input prefix or suffix
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Moved `.usa-input-group--[width]` classes out of the `usa-form` package.
+    summaryAdditional: These classes are now generated in the `usa-input-prefix-suffix` package and can be used without the `.usa-form` parent element.
+    affectsStyles: true
+    githubPr: 6232
+    githubRepo: uswds
+    versionUswds: 3.11.0
```

Comment:
**Suggestion:** It feels like it'd be a good idea to add a changelog to the Form component page since we're removing styles from this package. Do you think this would be worthwhile?

Reply:
**suggestion (non-blocking):** Additionally, part of me wants to hear the tone be more centered around the relevant component.

For input prefix, instead of saying
> Moved classes out of form

Putting input prefix first:
> Moved classes into `usa-input-prefix-package` from the `usa-form-package`

Paraphrased, but let me know what you think!

Reply:
Updated the summaries in [9b34c0f](https://github.com/uswds/uswds-site/pull/2991/commits/9b34c0fb8eaf0644da80b604ea94a3e96c832773)